### DISCUSSION TITLE
[Need Help] Do I need to use dbus / daemon for my use case

### DISCUSSION THREAD

**Discussion by ghost** [AsamK/signal-cli#752] [Q&A] [answered]
Hi,

Thanks again for your work, sorry for my n00b query.

Is there a guide or docs for installation in linux.

I've gotten the signal-cli to work fine.

I don't understand the need for dbus or daemon.

I only want to send messages via `signal-cli` to `--note-to-self`, hopefully with disappearing messages enabled, that's it, I have no use case to receive, like a send-msg-bot from my home server

Appreciate any help. Thanks.

**Comment by bublath** [comment, marked as answer]
Short answer: No need for daemon in your case
Longer answer: the Dbus Daemon mode is used to run signal-cli as daemon in the background. To communicate you would either have to write a program utilizing Dbus or use the dbus-send command line tool. The only advantage in your use case that I could see: In daemon mode signal-cli is already running (takes a while to come up depending on system performance), so it could be helpful in case you want it to react fast and/or need it often
